packages/core/src/types/vc-data-model.ts

```diff
@@ -34,6 +34,15 @@ export type CredentialSubject = {
   [x: string]: any
 }
 
+/**
+ * Expresses the credential status type (also referred to as the credential status method). 
+ * It is expected that the value will provide enough information to determine the current status of the credential 
+ * and that machine readable information needs to be retrievable from the URI. 
+ * For example, the object could contain a link to an external document noting whether or not the credential is suspended or revoked.
+ * @see https://www.w3.org/TR/vc-data-model/#status
```

Comment:
This explanation about the value does not refer to the `type` property which is a string to identify the status method.
The `value` being described here is the whole `credentialStatus: CredentialStatus` property of a VC.
It looks like the definition from https://www.w3.org/TR/vc-data-model/#status is broken and I raised an issue here: https://github.com/w3c/vc-data-model/issues/891

```suggestion
 *
 * @see https://www.w3.org/TR/vc-data-model/#status
```



BUT why do you think it's necessary to define this wrapper type here in the first place?
Why isn't `string` enough for the type property?

Reply:
It gives a more strong semantic for the field to avoid people misunderstanding what that string should represent. It's not necessary, but good to have.

About the explanation, you're right, I missed that.

And thank you for the issue to fix the VC data model status reference.